### ISSUE TITLE
pytest-subtests deprecation warning

### ISSUE THREAD

**Issue by reaperhulk** [pytest-dev/pytest-subtests#38]
pytest-subtests uses `CaptureFixture` in `pytest_subtests.py`, which causes the following warning on pytest (at least as of latest pytest):

```
PytestDeprecationWarning: A private pytest class or function was used.
```

**Comment by nicoddemus** [comment]
Thanks @reaperhulk,

Yeah unfortunately there's no way around it currently, I think. 

Hopefully we will be able to get this into pytest itself before 7.0.

**Comment by benbc** [comment]
Looks like this might be fixed by #52, although that doesn't reference this issue and isn't released yet.

**Comment by nicoddemus** [comment]
Thanks @alex, just released `0.7.0` fixing this.

**Comment by alex** [comment]
Thanks!
